test/indicators/flash-message.test.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import { mount } from 'enzyme'
+import { render, screen } from '@testing-library/react'
 import { FlashMessage } from '../../src/'
 
 test('FlashMessage only shows dismiss button when callback is provided', () => {
-  const wrapper = mount(<FlashMessage>Success!</FlashMessage>)
-  expect(wrapper.find('button.dismiss').exists()).toBe(false)
-  const dismissWrapper = mount(
+  render(<FlashMessage>Success!</FlashMessage>)
+  expect(screen.queryByRole('button')).not.toBeInTheDocument()
+  render(
```

Comment:
Should I be using `rerender` here?

Reply:
Not in this case. `rerender` should be used for testing the result of updating a prop, and it looks like we're just testing two different cases (when `onDismiss` is provided vs. not provided).